### PR TITLE
Suppress "put_" property setters in mdoc for UWP components.

### PR DESCRIPTION
**Pull request by IrinVoso**
UWP components use the prefix "put_" for property setter methods. mdoc does not currently recognize this prefix and thus creates an additional individual method for the property setter.

Internal bug: [76428](https://ceapex.visualstudio.com/Engineering/_workitems/edit/76428)

### CODE REVIEW

**Comment by dendeli-msft** [comment]
@joelmartinez does this also need a change in Mono.Cecil?

**Comment by joelmartinez** [comment]
@dendeli-msft no, this is mdoc-specific logic … basically we just need to ignore the "special" internally generated functions, and in this case, UWP simply deviated from the special names used by roslyn C# compiler :)

File: external/Test/readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The files in this directory are used as test inputs for explicit assembly testing.
```
Comment (joelmartinez) [inline external/Test/readme.md]: Great idea adding this readme

**Comment by joelmartinez** [comment]
@IrinVoso … I'll let you know when this makes it into a release. I hope to be able to compile this along with a few other changes into a release soon 🎉

**Pull request merged**